### ISSUE TITLE
settings.json isn't in the expected spot in Browser VSO

### ISSUE THREAD

**Issue by darthwalsh** [MicrosoftDocs/vsonline#337]
I use the [Settings Sync extension](https://marketplace.visualstudio.com/items?itemName=Shan.code-settings-sync) and syncing settings seems blocked on VSO's behavior.

It reported that it copied `settings.json` over my local VS Code `settings.json`, but when I use ">Preferences: Open Settings (JSON)" nothing has been updated. If I copy the path of that JSON file, VS Code says it is `/User/settings.json` which is not the same as is expected [in the vanilla docs](https://code.visualstudio.com/docs/getstarted/settings): `$HOME/.config/Code/User/settings.json`. If I try to access either of these paths from the terminal, they don't exist.

Please document what the expected path to edit `settings.json` should be for the Settings Sync extension.

(Also, after downloading my gist, it tried to install a few dozen extensions and many of them seemed to hang so I blacklisted them, but that's another issue.)

**Comment by jkeech** [comment]
With VSO (and I believe VS Code remote as well), settings are actually stored on the client side, not the remote side. In the browser in VSO, this translates to browser local storage. If the settings sync extension is writing a file on disk on the remote side, then it wouldn't work. But if it goes through VS Code's file system provider APIs to read/write settings, then it should work correctly.

VS Code is also working on built in settings sync which will work in VSO out of the box, which might be a different alternative is the Settings Sync extension doesn't work well with VS Code remote or VSO.

**Comment by darthwalsh** [comment]
Aha, thanks for the info. Sounds like it might be a extension bug then.
I'll wait to see if the upcoming built setting sync works for me, or follow
up with the extension.

--
War does not determine who is right.
War only determines who is left.

**Comment by lostintangent** [comment]
Yeah this is definitely a bug in the Settings Sync extension, so it would be awesome if you could file an issue with them 👍 I’m going to close this issue in the meantime, since we’re planning to add a built-in setting sync experience very soon. Thanks!
